bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerHandle.java

```diff
@@ -204,7 +204,7 @@ public void safeRun() {
     }
 
     protected void initializeExplicitLacFlushPolicy() {
-        if (!metadata.isClosed() && bk.getExplicitLacInterval() > 0) {
+        if (!metadata.isClosed() && !(this instanceof ReadOnlyLedgerHandle) && bk.getExplicitLacInterval() > 0) {
```

Comment:
yep, I missed this fix. Caught this while porting changes from community  to our codebase, but forgot to push it back to the community.